README.md

```diff
@@ -161,11 +192,27 @@ _Environment variable denoted in parentheses._
 * `--prefer-ts-exts` Re-order file extensions so that TypeScript imports are preferred (`TS_NODE_PREFER_TS_EXTS`, default: `false`)
 * `--log-error` Logs TypeScript errors to stderr instead of throwing exceptions (`TS_NODE_LOG_ERROR`, default: `false`)
 
-### Programmatic Only Options
+### Programmatic-only Options
+
+* `transformers` `_ts.CustomTransformers | ((p: _ts.Program) => _ts.CustomTransformers)`: An object with transformers or a function that accepts a program and returns an transformers object to pass to TypeScript. Function isn't available with `transpileOnly` flag
+* `readFile`: Custom TypeScript-compatible file reading function
+* `fileExists`: Custom TypeScript-compatible file existence function
+
+### Options via tsconfig.json
```

Comment:
I intentionally had this without a title in the `CLI and Programmatic Options` section as my understanding is that this only applies to those options (but not the CLI-only or programmatic-only options). Now we are mixing with the titles and sections *what* options are available and *how* to provide them. But given that neither approach is perfect, just do you what you think is best.